typescript/test/testdata/showcase/src/v1beta1/echo_client.ts.baseline

```diff
@@ -430,10 +467,20 @@ export class EchoClient {
     options = options || {};
     return this._innerApiCalls.wait(request, options, callback);
   }
-  /*
-  This is similar to the Expand method but instead of returning a stream of
- expanded words, this method returns a paged list of expanded words. 
-  */
+/**
+ *  This is similar to the Expand method but instead of returning a stream of
+ *  expanded words, this method returns a paged list of expanded words.
+ * 
+ * @param {string} request.content 
+ *  The string to expand. 
+ * 
+ * @param {int32} request.page_size 
```

Comment:
That won't work :(  There is no `int32` type in JavaScript. We need to have a real mapping.

The valid types are `object`, `number`, and `string`. If possible, it would be _great_ to use an actual message type instead of `object` (since we know it), but it can go into a separate PR if you prefer.

Reply:
upd. also `boolean` and `Buffer`

Reply:
Oh, I see, you were talking about the primitive type of js, working on this, thanks!